packages/mui-base/src/useFormControlUnstyledContext/useFormControlUnstyledContext.ts

```diff
@@ -1,5 +1,5 @@
 import * as React from 'react';
```

Comment:
This hook can stay in its previous location. It's not meant to be used to build components, but rather it's a utility wrapper around React's useContext.